htdocs/compta/facture/class/facture.class.php

```diff
@@ -199,6 +201,7 @@ class Facture extends CommonInvoice
 	public $fk_facture_source;
 	public $linked_objects = array();
 
+	public $date_valid;
```

Comment:
There is already a field ->date_validation that is used frequently.
So i think problem is where the code is using ->date_valid (should be date_validation) and not in the missing property.